Makefile

```diff
@@ -33,6 +33,11 @@ preproduction:
 	$(eval DEPLOY_ENV=preproduction)
 	echo https://check-childrens-barred-list-preproduction.test.teacherservices.cloud will be created in aks
 
+production:
```

Comment:
Once this is merged we can use this to run the new command:

`make production action-group-resources ACTION_GROUP_EMAIL=notificationemail@domain.com`

This will set up the resource group for monitoring.